config/olm/kustomization.yaml

```diff
@@ -10,7 +10,7 @@ namePrefix: kmm-operator-
 resources:
 - ../deploy
 labels:
-- includeSelectors: true
+- includeSelectors: false
```

Comment:
I think this file can be completely removed (in another, u/s first, preliminary PR). We don't use the CA from the node anymore - this was part of the old implementation when we pulled images using HTTP.

@yevgeny-shnaidman Do you see any reason to keep this one?